awscli/examples/iot-data/update-thing-shadow.rst

```diff
@@ -3,8 +3,9 @@
 The following ``update-thing-shadow`` example modifies the current state of the device shadow for the specified thing and saves it to the file ``output.txt``. ::
 
     aws iot-data update-thing-shadow \
+        --cli-binary-format legacy \
```

Comment:
I think for the examples where we are using the `--cli-binary-format` flag, we should convert these to just use `fileb://`. Mainly it feels weird to have examples where it uses a parameter value called `legacy` as `legacy` can be interpreted as something that should not be used. On a similar train of thought, I wonder if we should circle back on the value name here and to come up with a better name for the legacy binary format? Looking at this example, it seems completely reasonable to specify json in the command line and so it would seem strange to recommend using a value called `legacy` when we have no plans to deprecate it.

Reply:
Building on ideas of what to call it instead what are your thoughts on renaming `legacy` to `raw-base64`?

Reply:
The places I used `legacy` I did so because it's the most ergonomic option given the way the API is structured and the type of content that is trying to be passed. For this particular one I suppose I could move the contents to a file and then use `fileb://`.